css/project.css

```diff
@@ -5,30 +5,22 @@
 
 :root {
     --secondary-background: #FFFFFF;
+    --login-color: #2455f4;
 }
```

Comment:
use there var color

Reply:
You mean instead of login-color should i use something like secondary-color

Reply:
updated with css variable